src/java.base/share/classes/java/lang/ref/PhantomReference.java

```diff
@@ -60,6 +60,10 @@ public T get() {
         return null;
     }
 
+    /* Type-erased implementation of refersTo(). */
+    @Override
+    native final boolean refersTo0(Object o);
+
```

Comment:
How does the existence of this method help future intrinsification?
If it was called somewhere it would be clearer.  
Perhaps a comment now or later would help explain its function.